tests/store/index-level.spec.ts

```diff
@@ -1179,15 +1313,18 @@ describe('IndexLevel', () => {
   });
 
   describe('createCursorFromItem', () => {
-    it('throws if cursor value is not a number or boolean', async () => {
+    it('throws if cursor value is a boolean or an array', async () => {
+      // we can only sort by strings or numbers, so arrays or booleans should throw.
       const item: IndexedItem = {
         messageCid : 'message-cid',
         indexes    : {
-          sortProperty: true,
+          sortPropertyBool  : true,
```

Comment:
Not saying we should, just for my learning: why can't we sort by boolean?

Reply:
Sorting by boolean would effectively be the same as splitting the group into 2(true and false) and tie-breaking the rest (sorting by messageCid). It doesn't seem like a sortable property that has any meaningfulness behind it.

Reply:
I am not 100% convinced but also am not losing sleep over it, resolving.